pkg/machines/libvirt-common.js

```diff
@@ -905,15 +905,20 @@ export function updateDisk(domXml, diskTarget, readonly, shareable) {
     return tmp.innerHTML;
 }
 
-export function unknownConnectionName(action, libvirtServiceName) {
+export function unknownConnectionName(action, libvirtServiceName = undefined) {
```

Comment:
No need to set the explicit `= undefined`, that's the default anyway.

It's not clear what that function does, and below you call it several times with discarding the return value. I realize that it already existed before this PR, but a call like `unknownConnectionName(initInterfaces);` is rather unclear to read.